File: MainApp/Users/src/main/java/jvn/Users/model/UserTokenState.java
package jvn.Users.model;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
public class UserTokenState {

    private String accessToken;

    private Long expiresIn;

    private String refreshToken;

    public UserTokenState(String accessToken, long expiresIn, String refreshToken) {
        this.accessToken = accessToken;
        this.expiresIn = expiresIn;
        this.refreshToken = refreshToken;
    }

}

File: MainApp/ZuulAndFrontend/Frontend/src/app/model/userTokenState.ts
export class UserTokenState {
    accessToken: string;
    expireIn: number;
    refreshToken: string;

    constructor(accessToken: string, expireIn: number, refreshToken: string) {
        this.accessToken = accessToken;
        this.expireIn = expireIn;
        this.refreshToken = refreshToken;
    }
}